cfgov/templates/v1/demo/molecules/half_width_link_blob.html

```diff
@@ -0,0 +1,40 @@
+<!-- ==========================================================================
```

Comment:
moved all molecule/organisms to demo template folder reducing scope until jinja2 support is official released